File: src/main/java/com/moira/itda/global/entity/ImageFile.java
package com.moira.itda.global.entity;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.time.ZonedDateTime;

@Builder
@Getter
@Setter
public class ImageFile {
    private Long id;
    private String fileId;
    private String identifier;
    private String originalFileName;
    private String storedFileName;
    private Long size;
    private String fileUrl;
    private ZonedDateTime createdAt;
}

File: src/main/kotlin/com/moira/itda/global/entity/ImageFile.kt
package com.moira.itda.global.entity

import java.time.ZonedDateTime

data class ImageFile(
    val id: Long?,
    val fileId: String,
    val identifier: String,
    val originalFileName: String,
    val storedFileName: String,
    val size: Long,
    val fileUrl: String,
    val createdAt: ZonedDateTime
)